### ISSUE TITLE
Login to Epic Requires security check

### ISSUE THREAD

**Issue by RobertMeissner** [NikkelM/Steam-App-ID-Finder#22]
Steps:
- Logged in to transaction
- Copied the bearer token
- Ran getEpicGamesGames

Response 403, amongst others it returns

en: {
                challengeTitle: 'One More Step',
                challengeSubtitle: 'Please complete a security check to continue',
                sessionId: 'Session ID',
                ipAddress: 'IP Address',
            },

I temporarily deactivated 2FA completely, this did not resolve it. The token looks like "EPIC_BEARER_TOKEN=12314123adasd".

I am searching around but cannot find my bug/problem.

**Comment by NikkelM** [comment]
Hi @RobertMeissner, I was able to reproduce the issue. It seems that Epic Games has introduced some more security features (it looks like the response you got is a Cloudflare captcha) that blocks the tool from accessing the account data. I have added a workaround to the Readme [here](https://github.com/NikkelM/Steam-App-ID-Finder?tab=readme-ov-file#workaround-if-the-tool-throws-an-error) to get almost similar results through the browser console.
